tests/testdata/default-single-rack-single-node-prestop-dc.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: cassandra.datastax.com/v1beta1
+kind: CassandraDatacenter
+metadata:
+  name: dc2
+spec:
+  clusterName: cluster2
+  serverType: dse
+  serverVersion: "6.8.4"
+  managementApiAuth:
+    insecure: {}
+  size: 1
+  podTemplateSpec:
+    spec:
+      containers:
+        - name: "cassandra"
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sleep", "6000s"]
```

Comment:
This is to force the shutdown to take forever so the grace period timeout will kick-in.